addons/website/static/src/components/configurator/configurator.xml

```diff
@@ -154,12 +154,12 @@
         <div class="container-fluid pt-3 pb-2">
             <img class="ml-lg-5" style="height: 31px; width: 99px;" src="/website/static/src/img/odoo_logo.svg" title="Odoo Logo"/>
         </div>
-        <div class="o_configurator_screen_content container d-flex flex-grow-1 align-items-center">
-            <div class="m-auto">
-                <div class="o_configurator_typing_text o_configurator_show_fast pl-2">Add <b class="text-info">Pages</b> and <b class="text-warning">Features</b></div>
-                <h5 class="o_configurator_show_fast text-muted pl-2">You'll be able to create your pages later on.</h5>
-                <div class="page_feature_selection o_configurator_show">
-                    <div class="w-100 page_feature_selection container d-flex flex-wrap py2 py-lg-3">
+        <div class="o_configurator_screen_content overflow-hidden container d-flex flex-grow-1 align-items-center">
+            <div class="m-auto d-flex mh-100 flex-column">
+                <div class="o_configurator_typing_text o_configurator_show_fast">Add <b class="text-info">Pages</b> and <b class="text-warning">Features</b></div>
+                <h5 class="o_configurator_show_fast text-muted pb-lg-3">You'll be able to create your pages later on.</h5>
+                <div class="page_feature_selection o_configurator_show overflow-auto mt-lg-3 mx-n2">
+                    <div class="w-100 page_feature_selection d-flex flex-wrap">
```

Comment:
@detrouxdev I took your morning changes then push-forced your branch to remove your configurator commit and replace it with this change (as you can see, only some alignment/overflow/flex classes changes) and I think it achieves both the mobile and the small screen visual being ok to have only the features scrollable while keeping the same result in desktop. Could you confirm if it seems ok on your side? Thanks :+1:

Reply:
I think that's much closer to the original idea yeah ! You might still want to check with PDE if this is fine by him, there's one specific thing they pointed but that might not apply here since you achieved a better result : 
> The scrollbar should be on the full page, not on the div.
If they want the user to see all of our features before seeing the build my website button, then it might not be the ideal solution but IMO this is much better tho since we juste change classes.

Reply:
@pierre-pde What do you think?

Reply:
perfect for me, thanks @qsm-odoo  https://bit.ly/3laWLGx